core/domain/skill_services.py

```diff
@@ -839,6 +839,34 @@ def save_user_skill_mastery(user_skill_mastery):
     user_skill_mastery_model.put()
 
 
+def create_multi_user_skill_mastery(user_id, skill_ids, degrees_of_mastery):
+    """Creates the mastery of a user in multiple skills.
+
+    Args:
+        user_id: str. The user ID of the user.
+        skill_ids: list(str). Skill IDs of the requested skills.
+        degrees_of_mastery: list(float). The degrees of mastery of the user in
+            the requested skills.
+
+    Raises:
+        Exception: skill_ids and degrees_of_mastery lists have different
+            lengths.
+    """
+    if len(skill_ids) != len(degrees_of_mastery):
+        raise Exception(
+            'skill_ids and degrees_of_mastery should have the same length.')
+
+    user_skill_mastery_models = []
+
+    for skill_id, degree_of_mastery in zip(skill_ids, degrees_of_mastery):
+        user_skill_mastery_models.append(user_models.UserSkillMasteryModel(
```

Comment:
Don't we have any domain object for this model? (how do we validate the object before saving it to datastore?)

Reply:
There is domain object but it doesn't have a validation function yet. I will add it in the future if needed.